libminifi/include/properties/Properties.h

```diff
@@ -60,6 +60,9 @@ class Properties {
   // Get the config value
   bool get(std::string key, std::string &value);
 
+  // Get the config value
```

Comment:
This should document the alternate keys and why it is different than the above

Reply:
I think you're getting intermediate commits as I work through issues with another Apache person -- this one isn't quite ready for review. I'll close the PR in the meantime.

Reply:
sorry

Reply:
No apology, please. It's my fault for opening early!. I truly appreciate the input and hope that when I re-open it you have an opportunity to take a look!